worlds/sm64ex/__init__.py

```diff
@@ -104,7 +104,11 @@ def create_items(self):
         # 1Up Mushrooms
         self.multiworld.itempool += [self.create_item("1Up Mushroom") for i in range(0,self.filler_count)]
         # Power Stars
-        self.multiworld.itempool += [self.create_item("Power Star") for i in range(0,self.number_of_stars)]
+        star_range = self.number_of_stars
+        # Vanilla 100 Coin stars have to removed from the pool if other max star increasing options are active.
+        if (self.options.enable_coin_stars == "vanilla" and star_range >= 15):
```

Comment:
I feel like I'm missing something here. For one thing `star_range` can't be less than 15. But for another, even if it could, why would you want that to make _more_ stars exist?
```suggestion
        if self.options.enable_coin_stars == "vanilla":
```